examples/step-26/step-26.cc

```diff
@@ -260,16 +260,17 @@ namespace Step26
     laplace_matrix.reinit(sparsity_pattern);
     system_matrix.reinit(sparsity_pattern);
 
+    // do not pass the ConstraintMatrix as the last argument to the following
+    // functions, because our constraints are changing over time and are
+    // condense()-ed later.
     MatrixCreator::create_mass_matrix(dof_handler,
                                       QGauss<dim>(fe.degree+1),
                                       mass_matrix,
-                                      (const Function<dim> *)0,
-                                      constraints);
+                                      (const Function<dim> *)NULL);
     MatrixCreator::create_laplace_matrix(dof_handler,
                                          QGauss<dim>(fe.degree+1),
                                          laplace_matrix,
-                                         (const Function<dim> *)0,
-                                         constraints);
+                                         (const Function<dim> *)NULL);
```

Comment:
Both here and above, the optional argument `(const Function<dim> *)NULL` could now be omitted, since you are no longer passing the next one and the default is good...